dapps/digital-twin-detail/src/components/container/DataSetForm.ts

```
@@ -98,26 +103,45 @@ export default class DataSetFormComponent extends mixins(EvanComponent) {
       uiSpecs: {
         attr: {
           // translate it before, so the formular will use correct fallback translations
-          label: this.$t(`${this.i18nScope}.properties.${name}.label`, name),
-          placeholder: this.$t(`${this.i18nScope}.properties.${name}.placeholder`, ''),
+          label: this.isPrimitive ? false : this.$t(`${this.i18nScope}.properties.${name}.label`, name),
+          placeholder: this.isPrimitive
+            ? this.$t(`${this.i18nScope}.placeholder`, '')
+            : this.$t(`${this.i18nScope}.properties.${name}.placeholder`, ''),
         },
         type: 'input',
       },
-      value: this.value[name],
+      value: (this.isPrimitive ? this.value : this.value[name]) || '',
     };
 
     switch (type) {
+      case 'array': {
+        control.uiSpecs.type = 'json';
+        // test for valid json
+        try {
+          JSON.stringify(JSON.parse(control.value));
+        } catch (ex) {
+          control.value = [];
```

silently use empty array without error?